python/cuml/dask/datasets/blobs.py

```diff
@@ -89,6 +89,9 @@ def make_blobs(n_samples=100, n_features=2, centers=None, cluster_std=1.0,
         The order of the generated samples
     dtype : str, optional (default='float32')
         Dtype of the generated samples
+    workers : optional, list of strings
+        Dask addresses of workers to use for computation.
+        If None, all available Dask workers will be used.
```

Comment:
Can we provide a small example of how to find this and what a worker address might look like? (For example, it would help to add the way in we computed this list to the docs as well: `list(client.scheduler_info()['workers'].keys())` so that users know how to fill in this value. Adding a usage example would be nice as well- just to show a user how they might ask the dask client for these values and determine how they would pass in the ones they want to use.